src/site/layouts/page.drupal.liquid

```diff
@@ -26,6 +26,10 @@
         {% endif %}
 
         <article class="usa-content">
+          <div id="i18-link-wrapper" class="vads-u-display--none vads-u-margin-y--3 medium-screen:vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--3">
+            <i aria-hidden="true" class="fas fa-globe vads-u-color--primary"></i> 
+            <a class="i18-toggle" data-lang="en"></a>
+          </div>
```

Comment:
I don't understand this work very well but it would be nice if this followed our standard widget pattern. 

Instead of having all this in the template file you would just have -

```html
<article class="usa-content">
    <div data-widget-type="translation-button"></div>
    ...
```

Then add that `translation-button` into `src/applications/static-pages/widgetTypes.js`. The code at `src/applications/static-pages/caregiver-content-toggle/createCaregiverContentToggle.js` is an example. It also shows how you could be using React for this.

Reply:
(react might totally be overkill for this though)

Reply:
I'd just add that if there's an urgent need to ship this, and the code works as is, I would be OK with refactoring it later.

We should also do a better job of socializing/documenting the widget pattern.

Reply:
ok... i will def refactor this later given timelines. i'll create a GH issue for that for sure.

Reply:
although I didn't refactor the "Read this page in spanish/english" link b/c I thought react was overkill, I am using this approach for the new direction (to use a dropdown instead):
https://github.com/department-of-veterans-affairs/va.gov-team/issues/20212
and am using this component:
https://design.va.gov/storybook/?path=/docs/components-select--default